### DISCUSSION TITLE
Flet, privacy, telemetry, and Google Analytics

### DISCUSSION THREAD

**Discussion by DHBytes** [flet-dev/flet#1967] [General]
Flet is built with Flutter and according to the [Flutter docs](https://docs.flutter.dev/get-started/install/linux):

> The flutter tool uses Google Analytics to report feature usage statistics and send [crash reports](https://github.com/flutter/flutter/wiki/Flutter-CLI-crash-reporting). This data is used to help improve Flutter tools over time.
> 
> Flutter tool analytics are not sent on the very first run. To disable reporting, run flutter config --no-analytics. To display the current setting, use flutter config. If you opt out of analytics, an opt-out event is sent, and then no further information is sent by the Flutter tool.

Does a Flet app built in Python cause telemetry/analytics data to be sent during its execution? If so, how can I disable it?

Or did I misunderstand and the Flutter docs above refer to only directly developing in Flutter itself?

**Comment by wstevick** [comment]
I'd also like to know. I found this at the beginning of my digging, but I'll let you know if I figure it out.
